VolleyManagement.UI/spa/environments/environment.integrated.ts

```diff
@@ -0,0 +1,11 @@
+// The file contents for the current environment will overwrite these during build.
```

Comment:
So now you should definitely rename file to environment.vou.ts)))

Reply:
done